include/aspect/simulator_access.h

```diff
@@ -781,14 +781,32 @@ namespace aspect
        * constraints for the time step we are currently solving.
        */
       const ConstraintMatrix &
-      get_current_constraints() const;
+      get_current_constraints () const;
```

Comment:
This is actually unrelated to the PR. Please put it into a separate PR.

Reply:
See #2995.